code/game/objects/items/stunbaton.dm

```diff
@@ -63,9 +63,9 @@
 /obj/item/melee/baton/examine(mob/user)
 	..()
 	if(cell)
-		to_chat(user, "<span class='notice'>The baton is [round(cell.percent())]% charged.</span>")
+		to_chat(user, "<span class='notice'>The [src] is [round(cell.percent())]% charged.</span>")

```

Comment:
bring this lil change upstream if you can

Reply:
Done